### ISSUE TITLE
Question regarding the normalization values

### ISSUE THREAD

**Issue by fazelkhaha** [VSainteuf/pytorch-psetae#7]
Hi there,

I wonder how you calculated the Channel-wise normalization for each date? Mean and std over all the training dataset? To be more specific, let's say the first element of the tuple is a TxC array of mean values (array_1).  Is the first element of array_1 (array_1[0][0]) the mean value of all the pixels and all the training dataset for the first channel and first date?

I am a little bit confused.

**Comment by VSainteuf** [comment]
Hi fazelkhaha,
Yes it's exactly that!
The normalisation values are two arrays (one array of means and the other of stds) of shape TxC.
Each (_t,c_) position in the array of means corresponds to the average value  of channel _c_ of all the pixels of the date _t_.
And similarly for the std array.
